Makefile

```diff
@@ -1,9 +1,27 @@
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DELETE_ON_ERROR:
+
+# add easyjson source file targets here
+easyjson_sources := basm/dto.go x/xbasm/parse.go
+easyjson_generated := $(easyjson_sources:.go=_easyjson.go)
+
+# Rule to generate *_easyjson.go files
+%_easyjson.go: %.go
+	@easyjson $<
+
+# Generate all *_easyjson.go files
+generate: $(easyjson_generated)
```

Comment:
I updated the generate target to depend on the developer specified list of easyjson files. The
```makefile
%_easyjson.go: %.go
	@easyjson $<
```
rule tells the makefile how to convert from a .go file to _easyjson.go file.